x-pack/plugins/observability_solution/slo/common/locators/paths.ts

```diff
@@ -12,21 +12,24 @@ export const SLO_DETAIL_PATH = '/:sloId' as const;
 export const SLO_CREATE_PATH = '/create' as const;
 export const SLO_EDIT_PATH = '/edit/:sloId' as const;
 export const SLOS_OUTDATED_DEFINITIONS_PATH = '/outdated-definitions' as const;
+export const SLO_SETTINGS_PATH = '/settings' as const;
 
 export const paths = {
   slos: `${SLOS_BASE_PATH}${SLOS_PATH}`,
+  slosSettings: `${SLOS_BASE_PATH}${SLO_SETTINGS_PATH}`,
   slosWelcome: `${SLOS_BASE_PATH}${SLOS_WELCOME_PATH}`,
   slosOutdatedDefinitions: `${SLOS_BASE_PATH}${SLOS_OUTDATED_DEFINITIONS_PATH}`,
   sloCreate: `${SLOS_BASE_PATH}${SLO_CREATE_PATH}`,
   sloCreateWithEncodedForm: (encodedParams: string) =>
     `${SLOS_BASE_PATH}${SLO_CREATE_PATH}?_a=${encodedParams}`,
-  sloEdit: (sloId: string) => `${SLOS_BASE_PATH}${SLOS_PATH}/edit/${encodeURIComponent(sloId)}`,
+  sloEdit: (sloId: string) => `${SLOS_BASE_PATH}/edit/${encodeURIComponent(sloId)}`,
   sloEditWithEncodedForm: (sloId: string, encodedParams: string) =>
-    `${SLOS_BASE_PATH}${SLOS_PATH}/edit/${encodeURIComponent(sloId)}?_a=${encodedParams}`,
-  sloDetails: (sloId: string, instanceId?: string) =>
-    !!instanceId
-      ? `${SLOS_BASE_PATH}${SLOS_PATH}/${encodeURIComponent(sloId)}?instanceId=${encodeURIComponent(
-          instanceId
-        )}`
-      : `${SLOS_BASE_PATH}${SLOS_PATH}/${encodeURIComponent(sloId)}`,
+    `${SLOS_BASE_PATH}/edit/${encodeURIComponent(sloId)}?_a=${encodedParams}`,
+  sloDetails: (sloId: string, instanceId?: string, remoteName?: string) => {
+    const additionalArgs = remoteName ? `remoteName=${remoteName}` : '';
+    const sloPath = `${SLOS_BASE_PATH}${SLOS_PATH}/${encodeURIComponent(sloId)}`;
+    return !!instanceId
+      ? `${sloPath}?instanceId=${encodeURIComponent(instanceId)}&${additionalArgs}`
+      : `${sloPath}?${additionalArgs}`;
```

Comment:
We can leverage UrlSearchParams() to remove the need to self craft the query parameters URLs.
```
const qs = new URLSearchParams();
if (!!instanceId) qs.append("instanceId", instanceId);
if (!!remoteName) qs.append("remoteName", remoteName);
return `${SLOS_BASE_PATH}${SLOS_PATH}/${encodeURIComponent(sloId)}?${qs.toString()}`;
```